src/main/java/com/amazon/opendistroforelasticsearch/knn/plugin/script/KNNScoringUtil.java

```diff
@@ -207,4 +208,42 @@ public static float calculateHammingBit(BigInteger queryBigInteger, BigInteger i
     public static float calculateHammingBit(Long queryLong, Long inputLong) {
         return Long.bitCount(queryLong ^ inputLong);
     }
+
+    /**
+     * This method calculates L1 squared distance between query vector
+     * and input vector
+     *
+     * @param queryVector query vector
+     * @param inputVector input vector
+     * @return L1 score
+     */
+    public static float l1distance(float[] queryVector, float[] inputVector) {
```

Comment:
may be method name as l1, since we don't add distance to any other formula? @jmazanec15 what do you think?

Reply:
I think that is mistake in a comment, it should be l1 distance, and I think there is no l1squared distance, is it?

Reply:
Yeah. You are right. Sorry for typo. It is l2Squared